server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/rfc8621/RFC8621MethodsModule.java

```diff
@@ -172,13 +178,34 @@ JMAPRoutesHandler routesHandler(Set<JMAPRoutes> routes) {
     @Singleton
     @Named(InjectionKeys.RFC_8621)
     Authenticator provideAuthenticator(MetricFactory metricFactory,
-                                       Set<AuthenticationStrategy> authenticationStrategies) {
+                                       @Named("jmapRFC8621AuthenticationStrategies") Set<AuthenticationStrategy> authenticationStrategies) {
 
         return Authenticator.of(
             metricFactory,
             authenticationStrategies);
     }
 
+    @Provides
+    @Singleton
+    @Named("jmapRFC8621AuthenticationStrategies")
+    public Set<AuthenticationStrategy> provideAuthenticationStrategies(GuiceGenericLoader loader,
+                                                                       JmapRfc8621Configuration configuration,
+                                                                       JWTAuthenticationStrategy jwtAuthenticationStrategy,
+                                                                       BasicAuthenticationStrategy basicAuthenticationStrategy) {
```

Comment:
Please do not enforce the instanciation of authentication that are not strictly required. THis makes the startup slower than needed.

```suggestion
    public Set<AuthenticationStrategy> provideAuthenticationStrategies(GuiceGenericLoader loader,
                                                                       JmapRfc8621Configuration configuration) {
```